src/optimization/Combiner.h

```diff
@@ -53,7 +54,11 @@ namespace vc4c
 		 * Combines successive setting of the same flag (e.g. introduced by PHI-nodes)
 		 */
 		InstructionWalker combineSameFlags(const Module& module, Method& method, InstructionWalker it, const Configuration& config);
+
+		/*
+		 * Instruction scheduling for dual-issue, hide latency of memory load, etc...
```

Comment:
> Instruction scheduling for dual-issue

In `Values.h` and `IntermediateInstruction.h` you note things about combined instructions.
Should the instruction-scheduling take place before or after the instruction-combination?

I think the optimizer needs to schedule first, so you'll need to give the scheduling-pass a priority-value lower than the combination-pass in `Optimizer.cpp`.